### PR TITLE
Add test for uint256_mul_div_mod possible overflow

### PR DESCRIPTION
**Pull request by pefontana**
# Add test for uint256_mul_div_mod possible overflow

## Description

* Test the bug fixed in the PR #1320 
* Fix Makefile: The path to the `vm/src/tests/compare_vm_state.sh` script was outdated

## Checklist
- [x] Linked to Github Issue
- [ ] Unit tests added
- [x] Integration tests added.
- [ ] This change requires new documentation.
  - [ ] Documentation has been added/updated.
  - [ ] CHANGELOG has been updated.

### CODE REVIEW

File: vm/src/tests/cairo_run_test.rs [hunk 1]
```diff
@@ -335,7 +335,7 @@ fn dict_update() {
 #[cfg_attr(target_arch = "wasm32", wasm_bindgen_test)]
 fn uint256() {
     let program_data = include_bytes!("../../../cairo_programs/uint256.json");
-    run_program_simple_with_memory_holes(program_data.as_slice(), 3534);
+    run_program_simple_with_memory_holes(program_data.as_slice(), 3554);
```
Comment (Oppen) [inline vm/src/tests/cairo_run_test.rs:338]: Why did the number of steps change?

**Review by Oppen — APPROVED** [review]
